modular_skyrat/modules/morenarcotics/code/cocaine.dm

```diff
@@ -27,18 +27,16 @@
 	description = "A powerful stimulant extracted from coca leaves. Reduces stun times, but causes drowsiness and severe brain damage if overdosed."
 	reagent_state = LIQUID
 	color = "#ffffff"
-	overdose_threshold = 20
+	overdose_threshold = 10
```

Comment:
This ain't doing anything to its addictiveness. I would just flat out not touch overdose threshold because it's not needed